.gitignore

```diff
@@ -3,5 +3,5 @@ logs
 node_modules
 .idea/
 Gemfile.lock
-apps/**/translations/en
-public/
+apps/**/translations/**/default.json
+public
```

Comment:
we should have .DS_Store in here as it isn't ignored by git by default. Pretty sure npm ignores it though